docs/usage_parameters.adoc

```diff
@@ -216,10 +216,10 @@ _E.g.: ``ccaligner -wav tbbt.wav -srt tbbt.srt --quick-dict yes``_
 
 _E.g.: ``ccaligner -wav tbbt.wav -srt tbbt.srt -verbose info``_
 
-|`--display-recognised`
+|`-- display-recognized`
```

Comment:
It's a parameter, this can not be changed. Please undo all `recognized` to `recognised`.